continuousflex/protocols/protocol_mdspace.py

```diff
@@ -157,37 +159,42 @@ def rigidBodyAlignementStep(self):
             alignXMD.setValue(md.MDL_IMAGE, "", index)
 
         numpyArr2dcd(arrDCD, self._getExtraPath("coords.dcd"))
-        alignXMD.write(self.getAlignementprefix())
+        alignXMD.write(self.getTransformation())
 
     def updateAlignementStep(self):
 
-        if self.EMfitChoice.get() == EMFIT_VOLUMES:
-            if self._iter == 0:
-                inputSet = self.inputVolume.get()
-            else:
-                inputSet = self._createSetOfVolumes("inputSet")
-                readSetOfVolumes(self.getAlignementprefix(self._iter-1), inputSet)
-                inputSet.setSamplingRate(self.inputVolume.get().getSamplingRate())
-
-            inputAlignement = self._createSetOfVolumes("inputAlignement")
-            readSetOfVolumes(self.getAlignementprefix(), inputAlignement)
-            alignedSet = self._createSetOfVolumes("alignedSet")
+        # if self.EMfitChoice.get() == EMFIT_VOLUMES:
```

Comment:
If there is something to do here, better to keep a comment why this section is commented, otherwise, clean it.

Reply:
Yes